etc/kayobe/kolla-image-tags.yml

```diff
@@ -4,19 +4,19 @@
 # where the key is the OS distro and the value is the tag to deploy.
 kolla_image_tags:
   openstack:
-    rocky-9: 2023.1-rocky-9-20240423T125905
-    ubuntu-jammy: 2023.1-ubuntu-jammy-20240423T125905
+    rocky-9: 2023.1-rocky-9-20240621T104542
+    ubuntu-jammy: 2023.1-ubuntu-jammy-20240621T104542
   cloudkitty:
-    rocky-9: 2023.1-rocky-9-20240509T111619
-    ubuntu-jammy: 2023.1-ubuntu-jammy-20240509T111619
+    rocky-9: 2023.1-rocky-9-20240621T104542
```

Comment:
The tag for `openstack` is the default, so any like this that are identical can be removed